win32/getopt.h

```diff
@@ -731,9 +731,11 @@ int getopt_parse( int mode, getopt_std_args, ... )
 	 * specified for the `getopt_long' APIs.
 	 */
 	va_list refptr;
-	va_start( refptr, optstring );
+
 	struct option *longopts = va_arg( refptr, struct option * );
 	int *optindex = va_arg( refptr, int * );
+
+	va_start( refptr, optstring );
```

Comment:
You can't just move the `va_start` - the `va_start` call needs to be before the `va_arg` calls. You need to split the variable declarations of `longopts` and `optindex` from actually setting their value.